Derivs/src/derivs.cxx

```diff
@@ -27,6 +26,24 @@ calc_copy(const GF3D5<T> &gf, const GF3D5layout layout,
       });
 }
 
+template <int CI, int CJ, int CK, typename T>
+CCTK_ATTRIBUTE_NOINLINE void
+calc_copy(const vec<GF3D5<T>, dim> &gf, const GF3D5layout layout,
+          const GridDescBaseDevice &grid, const vec<GF3D2<const T>, dim> &gf0) {
+  for (int a = 0; a < dim; ++a)
+    calc_copy<CI, CJ, CK>(gf(a), layout, grid, gf0(a));
+};
```

Comment:
This seems to do more than "Extend 'Derivs' to cover 6th and 8th order finite difference" namely it introduces a `calc_copy` function (why this name) for a `vec<GF3D4<T>>` object. 

Nothing in the commit messages gives any indication that this is happening.

Reply:
`calc_copy` was there before, what I did is removing the extra argument `dx`, because it was not need.
The reason we need this is because spatial derivatives as stored in `GF3D5`, for consistency, we make a copy of var itself as `GF3D5`